static/bidder-info/compass.yaml

```diff
@@ -16,5 +16,8 @@ capabilities:
     - native
 userSync:
   redirect:
-    url: "https://sa-cs.deliverimp.com/pserver?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redir={{.RedirectURL}}"
+    url: "https://sa-cs.deliverimp.com/pserver?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&redir={{.RedirectURL}}"
+    userMacro: "[UID]"
+  iframe:
+    url: "https://sa-cs.deliverimp.com/pbserverIframe?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&pbserverUrl={{.RedirectURL}}"
```

Comment:
I'm getting a 404 when I try to use this URL:
`https://sa-cs.deliverimp.com/pbserverIframe?gdpr=0&gdpr_consent=CQJkOE3QJkOE3CuAAAENCZCAAP_AAH_AAAAAG5wAQG5gbnABAbmAAA.II7Nd_X__bX9n-_7_6ft0eY1f9_r37uQzDhfNs-8F3L_W_LwX32E7NF36tq4KmR4ku1bBIQNtHMnUDUmxaolVrzHsak2cpyNKJ_JkknsZe2dYGF9Pn9lD-YKZ7_5_9_f52T_9_9_-39z3_9f___dv_-__-vjf_599n_v9fV_78_Kf9______-____________8A&us_privacy=&gpp=&gpp_sid=&pbserverUrl=http%3A%2F%2Flocalhost%3A8000%2Fsetuid%3Fbidder%3Dcompass%26gdpr%3D0%26gdpr_consent%3DCQJkOE3QJkOE3CuAAAENCZCAAP_AAH_AAAAAG5wAQG5gbnABAbmAAA.II7Nd_X__bX9n-_7_6ft0eY1f9_r37uQzDhfNs-8F3L_W_LwX32E7NF36tq4KmR4ku1bBIQNtHMnUDUmxaolVrzHsak2cpyNKJ_JkknsZe2dYGF9Pn9lD-YKZ7_5_9_f52T_9_9_-39z3_9f___dv_-__-vjf_599n_v9fV_78_Kf9______-____________8A%26gpp%3D%26gpp_sid%3D%26f%3Db%26uid%3D%5BUID%5D`